apps/src/gamelab/GameLabJr.interpreted.js

```diff
@@ -265,8 +266,31 @@ function draw() {
   background(World.background_color || "white");
 
   if (World.frameCount > 1) {
-    // Run input events
-    for (var i=0; i<inputEvents.length; i++) {
+    for (let i = 0; i<sprites.length; i++) {
+      var sprite = sprites[i];
+
+      // Perform sprite behaviors
+      for (var j = 0; j<sprite.behavior_keys.length; j++) {
+        sprite.behaviors[sprite.behavior_keys[j]](sprite);
+      }
+
+      // Make sprites say things
+      if (sprite.things_to_say.length > 0) {
```

Comment:
Wondering if we should just nuke this for now?  We can always bring it back from the GitHub history if we need it.